sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -530,16 +541,20 @@ def start_bundle(self):
     self._counter_factory = counters.CounterFactory()
 
     # TODO(aaltay): Consider storing the serialized form as an optimization.
-    dofn = pickler.loads(pickler.dumps(transform.dofn))
+    dofn = (pickler.loads(pickler.dumps(transform.dofn))
+            if self._perform_dofn_pickle_test else transform.dofn)
```

Comment:
<!--new_thread; commit:3d6511ca1799a602673dbf658e25d6877931a944; resolved:0-->
Please add comment that this workaround is for SDF.